ui/app/AppLayouts/Browser/BrowserLayout.qml

```diff
@@ -488,11 +491,11 @@ Item {
                     source: currentWebView && currentWebView.icon ? currentWebView.icon : ""
                 }
                 focus: true
-                text: ""
+                text: currentWebView && currentWebView.url
```

Comment:
You probably need to put an `|| ""` just because QML is touchy and doesn't accept `undefined` as a "string"